### PR TITLE
Clarified versions available in deployment UI and CLI

### PR DESCRIPTION
**Pull request by sarahmaddox**
Clarified the Kubeflow versions available via the deployment UI and the CLI.

* Added clarifying text to the Pipelines tutorial (addressing issue https://github.com/kubeflow/website/issues/968) and also to the guide to the deployment UI.
* Removed shortcode `kf-stable-tag` as it's effectively the same as `kf-latest-version` in terms of the docs. It was causing muddles to have them both.
* Replaced the variable with a hard-coded `v0.5.0` in the GitOps doc, as that doc is not yet compatible with v0.6 anyway.

This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/981)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
Fixes https://github.com/kubeflow/website/issues/968

**Comment by sarahmaddox** [comment]
Preview:
* https://deploy-preview-981--competent-brattain-de2d6d.netlify.com/docs/gke/deploy/deploy-ui/
* https://deploy-preview-981--competent-brattain-de2d6d.netlify.com/docs/pipelines/tutorials/pipelines-tutorial/#deploy-kubeflow
* https://deploy-preview-981--competent-brattain-de2d6d.netlify.com/docs/use-cases/gitops-for-kubeflow/#create-kubeflow-deployment-repo

File: content/docs/gke/deploy/deploy-ui.md [hunk 1]
```diff
@@ -5,9 +5,12 @@ weight = 3
 +++
 
 This page provides instructions for using the Kubeflow deployment web app to
-deploy Kubeflow on GCP. If you prefer to use the
-command line, see the guide to 
+deploy Kubeflow on GCP. The deployment web app currently supports
+**Kubeflow {{% kf-deployment-ui-version %}}**.
+
+For more control over your deployment, see the guide to 
 [deployment using the CLI](/docs/gke/deploy/deploy-cli).
```
Comment (dwightjl) [inline content/docs/gke/deploy/deploy-ui.md]:
```suggestion
[deploy Kubeflow using the CLI](/docs/gke/deploy/deploy-cli).
```

File: content/docs/pipelines/tutorials/pipelines-tutorial.md [hunk 2]
```diff
@@ -128,22 +128,30 @@ Deploy Kubeflow on GCP:
   guide to [deploying Kubeflow on GCP](/docs/gke/deploy/), 
   taking note of the following:
 
+  * For the simplest deployment experience, use the Kubeflow deployment web app
```
Comment (dwightjl) [inline content/docs/pipelines/tutorials/pipelines-tutorial.md]:
```suggestion
  * If you want the most simple deployment experience, use the Kubeflow deployment web app
```

File: content/docs/pipelines/tutorials/pipelines-tutorial.md [hunk 3]
```diff
@@ -128,22 +128,30 @@ Deploy Kubeflow on GCP:
...
+    as described in the guide to 
+    [deployment using the UI](/docs/gke/deploy/deploy-ui/).
```
Comment (dwightjl) [inline content/docs/pipelines/tutorials/pipelines-tutorial.md]:
```suggestion
    [deploy Kubeflow using the UI](/docs/gke/deploy/deploy-ui/).
```

File: content/docs/pipelines/tutorials/pipelines-tutorial.md [hunk 4]
```diff
@@ -128,22 +128,30 @@ Deploy Kubeflow on GCP:
...
+    The deployment web app currently supports 
+    **Kubeflow {{% kf-deployment-ui-version %}}**.
+  * For more control
+    over the deployment, use the guide to 
+    [deployment using the CLI](/docs/gke/deploy/deploy-cli/).
```
Comment (dwightjl) [inline content/docs/pipelines/tutorials/pipelines-tutorial.md]:
```suggestion
    [deploy Kubeflow using the CLI](/docs/gke/deploy/deploy-cli/).
```

File: content/docs/gke/deploy/deploy-ui.md (hunk 1 above)
Comment (sarahmaddox) [reply content/docs/gke/deploy/deploy-ui.md]: Thanks for this suggestion! I don't think the suggested syntax works in this location in the sentence. We need a noun or noun phrase. So, either "deployment" or "deploying". I'll stick with what's there for now.

File: content/docs/pipelines/tutorials/pipelines-tutorial.md (hunk 3 above)
Comment (sarahmaddox) [reply content/docs/pipelines/tutorials/pipelines-tutorial.md]: Same comment as above.

File: content/docs/pipelines/tutorials/pipelines-tutorial.md (hunk 4 above)
Comment (sarahmaddox) [reply content/docs/pipelines/tutorials/pipelines-tutorial.md]: Same reply as above.

**Comment by sarahmaddox** [comment]
Thanks for the review @dwightjl! 

/assign @abhi-g

**Comment by IronPan** [comment]
/lgtm
/approve
What is the plan for supporting 0.6 in deployment UI? Hopefully this inconsistency will be gone soon.

**Pull request merged**